### PR TITLE
ACCESS-OM2 Model Run: Added top level "About" section

### LINKED ISSUE #680: Standard links from model run docs
## Description

We need some standard links/info/badges at the top of all model run docs for all models:
- license
- latest version
- GitHub repo

We also need a standard "About" introduction that gives a little bit of context, what the page is for, and for whom. A link to the release notes should also be included.

## Why

Standard information and placing of that information is important to ensure the information provided is consistent and easily discoverable.

Links back to the model description is important as users can come to the model run page through different paths, e.g. directly from a web search.

## Close Criteria

A merged PR with the changes.

### PR DESCRIPTION
**Pull request by aidanheerdegen**
## Description

Added top level About section.

Fixes #680

### CODE REVIEW

**Review by aidanheerdegen — COMMENTED** [review]
I added the GitHub icon at the top to see how it would render. I wondered if there was value in having that icon in a standard page header location (not the top level header)

**Comment by atteggiani** [comment]
Thank you for this @aidanheerdegen. 
I think it's a great idea.

I changed the base to `development` instead of main. 
I am going to review it next week (mainly change link to markdown and re-style the button).

**Comment by atteggiani** [comment]
@aidanheerdegen 

I saw that you created your branch from `main`.
Before my review, can you please rebase from `development`?

**Comment by aidanheerdegen** [comment]
> Before my review, can you please rebase from `development`?

Done.

File: docs/models/run-a-model/run-access-om.md [hunk 1]
```diff
@@ -5,20 +5,30 @@
 [payu]: https://github.com/payu-org/payu
 [model components]: https://access-hive.org.au/models/configurations/access-om/#model-components
 
+[:fontawesome-brands-github: GitHub](https://github.com/ACCESS-NRI/access-om2-configs){ .md-button }
```
Comment (atteggiani) [inline docs/models/run-a-model/run-access-om.md:8]:
```suggestion
[:fontawesome-brands-github:{: class="twemoji icon-before-text"} GitHub](https://github.com/ACCESS-NRI/access-om2-configs){: class="text-card"}
```

For now I would style the button as the one at the end of the [ACCESS-OM2 overview page](https://access-hive.org.au/pr-preview/pr-681/models/configurations/access-om/#access-om2)

Also, the rendering after merging #679 will not be exactly as the one shown in this PR preview.
The `icon-before-text` class does not exist in this branch, and it gives the GitHub icon right-margin so it's more spaced from the text.

The final rendering will be this:
Screenshot 2024-05-06 at 10 13 32 AM

File: docs/models/run-a-model/run-access-om.md [hunk 2]
```diff
@@ -5,20 +5,30 @@
...
+
 # Run {{ model }}
 !!! info
     In this documentation, the same code is sometimes shown in a highlighted code-block and also in a simulated terminal.<br>
     The code-blocks show the commands to be run in a terminal. They can be easily copied by clicking on the icon over the right side of the code block.<br>
     The simulated terminals are examples of the output to expect when the commands are run. Sometimes they might slightly differ from the real outputs.
 
+## About
+
+{{ model }} is an Ocean Sea-Ice model. More information is available in the [model overview](https://access-hive.org.au/models/configurations/access-om/).
```
Comment (atteggiani) [inline docs/models/run-a-model/run-access-om.md:18]:
```suggestion
{{ model }} is an Ocean Sea-Ice model. More information is available in the [{{ model }} overview](https://access-hive.org.au/models/configurations/access-om/#access-om2).
```

File: docs/models/run-a-model/run-access-om.md [hunk 3]
```diff
@@ -5,20 +5,30 @@
...
+
+If you are unsure whether {{ model }} is the right choice for your experiment, take a look at the overview of [ACCESS Models](https://access-hive.org.au/models).
+
+All Model configurations are open source, licensed under [CC BY 4.0](https://creativecommons.org/licenses/by/4.0/?ref=chooser-v1")<img style="height:22px!important;margin-left:3px;vertical-align:text-bottom;" src="https://mirrors.creativecommons.org/presskit/icons/cc.svg?ref=chooser-v1" alt=""><img style="height:22px!important;margin-left:3px;vertical-align:text-bottom;" src="https://mirrors.creativecommons.org/presskit/icons/by.svg?ref=chooser-v1" alt=""> and [available on GitHub](https://github.com/ACCESS-NRI/access-om2-configs).
```
Comment (atteggiani) [inline docs/models/run-a-model/run-access-om.md:22]:
```suggestion
All Model configurations are open source, licensed under [CC BY 4.0](https://creativecommons.org/licenses/by/4.0/?ref=chooser-v1")![CC icon](https://mirrors.creativecommons.org/presskit/icons/cc.svg?ref=chooser-v1){: style="height:1em;margin-left:0.2em;vertical-align:text-top;"}![BY icon](https://mirrors.creativecommons.org/presskit/icons/by.svg?ref=chooser-v1){: style="height:1em;margin-left:0.2em;vertical-align:text-top;"} and available on [ACCESS-NRI GitHub](https://github.com/ACCESS-NRI/access-om2-configs).
```

**Review by atteggiani — CHANGES_REQUESTED** [review]
Overall pretty good. 
I made some changes to the button styling, and changed everything to Markdown syntax.

**Review by atteggiani — APPROVED** [review]

**Comment by aidanheerdegen** [comment]
@paigem this is the PR to update the OM2 model run docs

**Comment by paigem** [comment]
This looks great @aidanheerdegen and @atteggiani! I'm going to add a couple small comments here, since I don't want to slow this process down as I remember how to contribute to an open PR (sorry - I guess I need to review some GitHub training! 😅).

- I like the GitHub button at the top, but is it possible to change the text in side to say which GitHub repo it points to (looks like the ACCESS-OM2 configurations repo)? Within a docs environment, I often assume the GitHub button would take me to the docs repo where I could suggest updates, so clarifying that link would be nice.
- I know this information is elsewhere, but I'd probably add one sentence at the top for new/international folks along the lines of the following: "The instructions below outline how to run the model using ACCESS-NRI's software deployment pipeline, specifically designed to run on the [National Computating Infrastructure's (NCI's)](https://nci.org.au/about-us/who-we-are) Gadi supercomputer." The two points I think are worth mentioning:
  - This documentation refers to the ACCESS-NRI release of OM2 (not the legacy one).
  - This is specifically for running on Gadi.
- Kelsey also suggested adding a link to the model code. I actually couldn't find the model code - it looks like it's all neatly packaged in [this repo](https://github.com/ACCESS-NRI/ACCESS-OM2/tree/main), where cloning this repo and running a few commands will run the model on Gadi. Is the model code available somewhere, and is that something worth linking to?

File: docs/models/run-a-model/run-access-om.md [hunk 4]
```diff
@@ -5,20 +5,30 @@
...
 # Run {{ model }}
 !!! info
     In this documentation, the same code is sometimes shown in a highlighted code-block and also in a simulated terminal.<br>
     The code-blocks show the commands to be run in a terminal. They can be easily copied by clicking on the icon over the right side of the code block.<br>
     The simulated terminals are examples of the output to expect when the commands are run. Sometimes they might slightly differ from the real outputs.
 
+## About
+
+{{ model }} is an Ocean Sea-Ice model. More information is available in the [{{ model }} overview](https://access-hive.org.au/models/configurations/access-om/#access-om2).
+
```
Comment (paigem) [inline docs/models/run-a-model/run-access-om.md:19]:
```suggestion

The instructions below outline how to run the model using ACCESS-NRI's software deployment pipeline, specifically designed to run on the [National Computating Infrastructure's (NCI's)](https://nci.org.au/about-us/who-we-are) Gadi supercomputer.

```

**Comment by paigem** [comment]
Thanks @aidanheerdegen!

> But it could be more explicit. e.g. as a table, with a column being a link to the code repo.

Thanks - you're right that it makes sense now that we just link out to each component's code. 🤦‍♀️ What might make it slightly clearer is to, instead of having the links on the name of the models (I thought those links would go to information about the model, rather than model code), to have in parentheses after each model is listed "(link to MOM5 code)", for example, to make it clear where the links direct to. But, I think this is minor and not worth worrying about now - the information is there and quite readable as is.

**Comment by aidanheerdegen** [comment]
> What might make it slightly clearer is to, instead of having the links on the name of the models (I thought those links would go to information about the model, rather than model code), to have in parentheses after each model is listed "(link to MOM5 code)", for example, to make it clear where the links direct to

I agree. It's what I was getting at with the table, but I explained badly. Feel free to make an issue to improve the README in that repo. It won't get fixed otherwise.

**Comment by atteggiani** [comment]
Thank you @paigem for the suggestions. 

I committed your suggestions
>The instructions below outline how to run the model using ACCESS-NRI's software deployment pipeline, specifically designed to run on the [National Computating Infrastructure's (NCI's)](https://nci.org.au/about-us/who-we-are) Gadi supercomputer.

with minor modifications:
>The instructions below outline how to run the ACCESS-OM2 using ACCESS-NRI's software deployment pipeline, specifically designed to run on the [National Computating Infrastructure (NCI)](https://nci.org.au/about-us/who-we-are) supercomputer [_Gadi_](https://opus.nci.org.au/display/Help/0.+Welcome+to+Gadi#id-0.WelcometoGadi-Overview).

I also changed the Github button text to "ACCESS-OM2 configurations".

If no more modifications are required, I will merge this PR around 3:30pm today.

Thank you!
@aidanheerdegen

**Comment by paigem** [comment]
This all looks great @atteggiani! All my comments have been addressed - thanks! 😊

**Comment by atteggiani** [comment]
Also rebased with `development`

**Pull request merged**